src/Shared/Recruit.Vacancies.Client/Infrastructure/EventHandlers/UpdateFAAOnApplicationReviewed.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Events;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Services.FAA;
+using MediatR;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.EventHandlers
+{
+    public class UpdateFaaOnApplicationReviewed :
+        INotificationHandler<ApplicationReviewedEvent>
+    {
+        private readonly IFaaService _faaService;
+
+        public UpdateFaaOnApplicationReviewed(IFaaService faaService)
+        {
+            _faaService = faaService;
+        }
+
+        public Task Handle(ApplicationReviewedEvent notification, CancellationToken cancellationToken)
+        {
+            var message = new FaaApplicationStatusSummary
+            {
+                VacancyReference = (int)notification.VacancyReference,
+                UnsuccessfulReason = notification.CandidateFeedback,
+                ApplicationStatus = GetApplicationStatus(notification.Status),
+                CandidateId = notification.CandidateId,
+                IsRecruitVacancy = true
+            };
+
+            return _faaService.PublishApplicationStatusSummaryAsync(message);
+        }
+
+        private FaaApplicationStatus GetApplicationStatus(ApplicationReviewStatus status)
+        {
+            switch (status)
+            {

```

Comment:
`New` is a valid option.  When an ApplicationReview is first created it is in `New` status.  Its is subsequently made `Successful` or `Unsuccessful`. In the future it could other statuses e.g. `Interview` or `Declined` for example.

I've used a switch and NotImplementedException as I think we'll be adding new statuses and the handling of those new statuses needs to be implemented.

Reply:
I'll change this as FAA will only care about Successful/Unsuccessful so we don't need to care about other statuses